tpchgen-cli/src/main.rs

```diff
@@ -61,137 +68,323 @@ enum Table {
     LineItem,
 }
 
+impl Table {
+    fn name(&self) -> &'static str {
+        match self {
+            Table::Nation => "nation",
+            Table::Region => "region",
+            Table::Part => "part",
+            Table::Supplier => "supplier",
+            Table::PartSupp => "partsupp",
+            Table::Customer => "customer",
+            Table::Orders => "orders",
+            Table::LineItem => "lineitem",
+        }
+    }
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
+enum OutputFormat {
```

Comment:
I added an output format enum and then made the various generation functions methods on `Cli`